napari/components/viewer_model.py

```diff
@@ -587,11 +587,18 @@ def add_points(
         self,
         data=None,
         *,
+        properties=None,
         symbol='o',
         size=10,
         edge_width=1,
         edge_color='black',
+        edge_color_cycle=None,
+        edge_color_colormap='viridis',
```

Comment:
could this just be `edge_colormap` and `edge_contrast_limits` - might be a little easier to read

Reply:
I think this makes sense. I'll make the change now!